src/Kunstmaan/MediaBundle/Helper/ExtensionGuesserFactoryInterface.php

```diff
@@ -2,12 +2,19 @@
 
 namespace Kunstmaan\MediaBundle\Helper;
 
-use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface;
 
+use Symfony\Component\Mime\MimeTypes;
+use Symfony\Component\Mime\MimeTypesInterface;
+
+@trigger_error(sprintf('The "%s" class is deprecated since KunstmaanMediaBundle 5.6, use "%s" instead.', ExtensionGuesserFactoryInterface::class, MimeTypes::class), E_USER_DEPRECATED);
+
+/**
+ * @deprecated this class is deprecated since KunstmaanMediaBundle 5.6 and will be removed in KunstmaanMediaBundle 6.0
+ */
 interface ExtensionGuesserFactoryInterface
 {
     /**
-     * @return ExtensionGuesserInterface
+     * @return MimeTypesInterface
      */
-    public function get();
+    public function get(): MimeTypesInterface;
```

Comment:
There should be no changes in this file as it is deprecated. Also this change is a BC break